package.json

```diff
@@ -29,7 +30,7 @@
     "@types/node": "^14.0.5",
     "@zemse/solidity-flattener": "^1.5.1",
     "eth-proof": "^2.1.4",
-    "ethers": "^5.0.19",
+    "ethers": "5.0.19",
```

Comment:
This cange was intentional only 

just because I got some errors mentioned below 
![image](https://user-images.githubusercontent.com/45527699/101522334-120e8900-39ad-11eb-888e-07e6ab05b55f.png)

Reply:
The ethers change is fine.